src/main/java/org/myrobotlab/framework/repo/IvyWrapper.java

```diff
@@ -256,21 +259,31 @@ synchronized public void install(String location, String[] serviceTypes) {
 
       // TODO: this breaks for me! please review why this needed to be commented
       // out.
+      // Ivy ivy = Ivy.newInstance(); <-- for future 2.5.x release
+      // ivy.getLoggerEngine().pushLogger(new IvyWrapperLogger(Message.MSG_INFO)); <-- for future 2.5.x release
       Main.setLogger(new IvyWrapperLogger(Message.MSG_INFO));
       ResolveReport report = Main.run(cmd);
 
-      // if no errors -
+      // if no errors -h
       // mark "service" as installed
       // mark all libraries as installed
 
-      List<?> err = report.getAllProblemMessages();
+      List<?> err= report.getAllProblemMessages();
 
+      boolean error = false;
       if (err.size() > 0) {
         for (int i = 0; i < err.size(); ++i) {
           String errStr = err.get(i).toString();
+          if (!errStr.startsWith("WARN:  symlinkmass")){

```

Comment:
This seems really specific..  are there other sorts of warning messages that we might miss?

Reply:
This is in response to this - https://issues.apache.org/jira/projects/IVY/issues/IVY-1609?filter=allissues
Which was improper errors coming back - We check errors, but this is not an error (shouldn't even be a warning if not invoked).  I'll comment it out, since there were a couple serious blockers to move to ant ivy 2.5.x